main/src/cgeo/geocaching/maps/mapsforge/MapsforgeMapView.java

```diff
@@ -183,6 +183,12 @@ public int getMapZoomLevel() {
         return getMapPosition().getZoomLevel() + 1;
     }
 
+    @Override
+    public void zoomToBounds(final Viewport bounds, GeoPointImpl center) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter center should be final.](https://app.codacy.com/manual/cgeo/cgeo/pullRequest?prid=4742102)

Reply:
Hmph, I thought I had called CheckStyle for all changed files... Should be somehow integrated in the local build process automatically. Anyway: Thanks for catching.